### PR TITLE
Add pyboard v1.1 schematic link

### LINKED ISSUE #4558: Docs don't link to Pyboard v1.1 schematic
See https://docs.micropython.org/en/latest/pyboard/general.html#the-pyboard-hardware

Only the v1.0 schematics are linked, but all new purchases will be v1.1.

### PR DESCRIPTION
**Pull request by johnthagen**
Fixes #4558.

Is there a single PDF that also has the v1.1 layouts?

### CODE REVIEW

File: docs/pyboard/hardware/index.rst [hunk 1]
```diff
@@ -3,9 +3,12 @@ The pyboard hardware
 
 For the pyboard:
 
-* `PYBv1.0 schematics and layout <http://micropython.org/resources/PYBv10b.pdf>`_ (2.4MiB PDF)
-* `PYBv1.0 metric dimensions <http://micropython.org/resources/PYBv10b-metric-dimensions.pdf>`_ (360KiB PDF)
-* `PYBv1.0 imperial dimensions <http://micropython.org/resources/PYBv10b-imperial-dimensions.pdf>`_ (360KiB PDF)
+* v1.1
+    * `PYBv1.1 schematics <https://micropython.org/resources/PYBv11-schematics.pdf>`_ (261KiB PDF)
```
Comment (dpgeorge) [inline docs/pyboard/hardware/index.rst]: Can you please update this to PYBv11.pdf?  And a description could be "schematics, layout and dimensions".

Comment (johnthagen) [reply docs/pyboard/hardware/index.rst]: Fixed in 9827cf9e40a2ffb2aef8efbc21d076de7be72553

**Comment by johnthagen** [comment]
@dpgeorge This PR (and #4587) should be good to go know.

**Comment by dpgeorge** [comment]
Thanks, merged in d390ad90534d35d31d891ad24f723675e4b077f5

**Pull request closed without merging**